backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,22 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
```

Comment:
@jhawthorn  and @cbrunsdon  expressed negativity towards `Spree::Backend::Callbacks`, so I've avoided using one in the controller in this case